com.unity.netcode.gameobjects/Runtime/Configuration/NetworkPrefabs.cs

```diff
@@ -0,0 +1,283 @@
+using System;
+using System.Collections.Generic;
+using System.Text;
+using UnityEngine;
+
+namespace Unity.Netcode
+{
+    [Serializable]
+    public class NetworkPrefabs
+    {
+        /// <summary>
+        /// Edit-time scripted object containing a list of NetworkPrefabs.
+        /// </summary>
+        /// <remarks>
+        /// This field can be null if no prefabs are pre-configured.
+        /// Runtime usages of <see cref="NetworkPrefabs"/> should not depend on this edit-time field for execution.
+        /// </remarks>
+        [SerializeField]
+        public NetworkPrefabsList NetworkPrefabsList;
+
+        /// <summary>
+        /// This dictionary provides a quick way to check and see if a NetworkPrefab has a NetworkPrefab override.
+        /// Generated at runtime and OnValidate
+        /// </summary>
+        [NonSerialized]
+        public Dictionary<uint, NetworkPrefab> NetworkPrefabOverrideLinks = new Dictionary<uint, NetworkPrefab>();
+
+        [NonSerialized]
+        public Dictionary<uint, uint> OverrideToNetworkPrefab = new Dictionary<uint, uint>();
+
+        public IReadOnlyList<NetworkPrefab> Prefabs => m_Prefabs;
+
+        [NonSerialized]
+        private List<NetworkPrefab> m_Prefabs = new List<NetworkPrefab>();
+
+        private void AddTriggeredByNetworkPrefabList(NetworkPrefab networkPrefab)
+        {
+            if (AddPrefabRegistration(networkPrefab))
+            {
+                m_Prefabs.Add(networkPrefab);
+            }
+        }
+
+        private void RemoveTriggeredByNetworkPrefabList(NetworkPrefab networkPrefab)
+        {
+            m_Prefabs.Remove(networkPrefab);
+        }
+
+        ~NetworkPrefabs()
+        {
+            if (NetworkPrefabsList)
+            {
+                NetworkPrefabsList.OnAdd -= AddTriggeredByNetworkPrefabList;
+                NetworkPrefabsList.OnRemove -= RemoveTriggeredByNetworkPrefabList;
+            }
+        }
+
+        /// <summary>
+        /// Processes the <see cref="NetworkPrefabsList"/> if one is present for use during runtime execution,
```

Comment:
I think it might be the xml notation that's confusing you.

"Processes the NetworkPrefabsList if one is present"

To me that makes sense ("Processes the NetworkPrefabsList if a NetworkPrefabsList is present")

Reply:
Oh, I think you're right.